### ISSUE TITLE
Newlines are removed from multi-line strings

### ISSUE THREAD

**Issue by jochenberger** [nvuillam/npm-groovy-lint#252]
When formatting Groovy files, line breaks are removed from strings.

```
$ cat test.groovy 
def s = '''




'''
$ npm-groovy-lint --format --verbose test.groovy 
  2     fixed    File test.groovy has consecutive blank lines  ConsecutiveBlankLines   

npm-groovy-lint results in  linted files:
┌─────────┬───────────┬─────────────┬─────────────┬─────────────────┐
│ (index) │ Severity  │ Total found │ Total fixed │ Total remaining │
├─────────┼───────────┼─────────────┼─────────────┼─────────────────┤
│    0    │  'Error'  │      0      │      0      │        0        │
│    1    │ 'Warning' │      0      │      0      │        0        │
│    2    │  'Info'   │      1      │      1      │        0        │
└─────────┴───────────┴─────────────┴─────────────┴─────────────────┘
Failure: 0 error(s) have been found 
 0 warning(s) have been found 
 1 info(s) have been found
$ cat test.groovy 
def s = '''

'''

**Comment by jochenberger** [comment]
That's probably an upstream issue: https://github.com/CodeNarc/CodeNarc/issues/720

**Comment by nvuillam** [comment]
As you notified in https://github.com/CodeNarc/CodeNarc/issues/720, the error comes from the false positive trigger by CodeNarc
Once it will be solved in CodeNarc and when they'll release a new version, it will be upgraded in npm-groovy-lint and the problem will disappear :)

**Comment by jochenberger** [comment]
Not sure if an upstream fix will make this issue go away. IMHO, *this* issue is about the formatter breaking the code, or at least changing its behavior, which a mere formatter shouldn't.
Maybe the formatter should not be enabled by default.
